### PR TITLE
Update links and credits in README

### PR DESCRIPTION
**Pull request by jferris**
* Link to contributing, license guides
* Update credit section for newer thoughtbot references

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,20 +130,36 @@ page.driver.header 'Referer', 'https://www.thoughtbot.com'
...
+![thoughtbot](https://thoughtbot.com/logo.png)
 
+capybara-webkit is maintained and funded by thoughtbot, inc.
 The names and logos for thoughtbot are trademarks of thoughtbot, inc.
 
-License
--------
+We love open source software!
+See [our other projects][community].
+We are [available for hire][hire].
```
Comment (croaky) [inline README.md]: I liked @tysongach's suggested rewording on Bourbon:

> We love open source software! See our other projects or hire us to help build your product.

Comment (croaky) [inline README.md]: I ended up going with:

```
See [our other projects][community] or
[hire us][hire] to design, develop, and grow your product.
```

File: README.md [hunk 2]
```diff
@@ -130,20 +130,36 @@ page.driver.header 'Referer', 'https://www.thoughtbot.com'
...
 
-The capybara WebKit driver is maintained by Joe Ferris and Matt Horan. It was written by [thoughtbot, inc](http://thoughtbot.com/community) with the help of numerous [contributions from the open source community](https://github.com/thoughtbot/capybara-webkit/contributors).
+The capybara WebKit driver is maintained by Joe Ferris and Matt Horan.
 
-Code for rendering the current webpage to a PNG is borrowed from Phantom.js' implementation.
+Code for rendering the current webpage to a PNG is borrowed from Phantom.js'
+implementation.
 
-![thoughtbot](http://thoughtbot.com/images/tm/logo.png)
+![thoughtbot](https://thoughtbot.com/logo.png)
```
Comment (croaky) [inline README.md]: This logo is out of date. New version is http://thoughtbot.github.io/images/signature.svg

File: README.md [hunk 3]
```diff
@@ -130,20 +130,36 @@ page.driver.header 'Referer', 'https://www.thoughtbot.com'
...
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/capybara-webkit/graphs/contributors
+
+License
+-------
+
+capybara-webkit is Copyright (c) 2010-2015 thoughtbot, inc. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
```
Comment (croaky) [inline README.md]: License file could be updated through 2015. Currently says 2012.

**Comment by croaky** [comment]
Good to merge.

File: README.md (hunk 2 above)
Comment (jferris) [inline README.md]: Should we update `thoughtbot.com/logo.png` instead of changing all the references to it?

File: README.md (hunk 1 above)
Comment (jferris) [inline README.md]: Updated.

File: README.md (hunk 3 above)
Comment (jferris) [inline README.md]: Updated.

File: README.md (hunk 2 above)
Comment (croaky) [inline README.md]: @jferris Ha, yes. Good call.

**Pull request merged**